internal/chatgpt/request.go

```diff
@@ -36,46 +33,16 @@ var (
 		tls_client.WithInsecureSkipVerify(),
 	}
 	client, _         = tls_client.NewHttpClient(tls_client.NewNoopLogger(), options...)
-	http_proxy        = os.Getenv("http_proxy")
 	API_REVERSE_PROXY = os.Getenv("API_REVERSE_PROXY")
 )
 
 func init() {
-	// Check for proxies.txt
-	if _, err := os.Stat("proxies.txt"); err == nil {
-		// Each line is a proxy, put in proxies array
-		file, _ := os.Open("proxies.txt")
-		defer file.Close()
-		scanner := bufio.NewScanner(file)
-		for scanner.Scan() {
-			// Split line by :
-			proxy := scanner.Text()
-			proxy_parts := strings.Split(proxy, ":")
-			if len(proxy_parts) == 2 {
-				proxy = "socks5://" + proxy
-			} else if len(proxy_parts) == 4 {
-				proxy = "socks5://" + proxy_parts[2] + ":" + proxy_parts[3] + "@" + proxy_parts[0] + ":" + proxy_parts[1]
-			} else {
-				continue
-			}
-			proxies = append(proxies, proxy)
-		}
-	}
```

Comment:
this is for iterating through multiple proxies

Reply:
New checkproxy func do this too, just no need to check the proxy url format. example
```
socks5://myproxy:8080
socks5://user:pass@myproxy:8080
```